File: 240602/두 개의 동일한 수열/two-equal-series.js
let fs = require("fs");

let input = fs.readFileSync(0).toString().split("\n");
let arr1 = input[1].split(" ").map(Number).sort((a , b) => a - b);
let arr2 = input[2].split(" ").map(Number).sort((a , b) => a - b);
let flag = true;

for(let i = 0; i < arr1.length; i++){
    if(arr1[i] !== arr2[i]) flag = false;
}

if(flag) console.log("Yes");
else console.log("No");

File: 240115/두 개의 동일한 수열/two-equal-series.cpp
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    int n;
    cin >> n;
    int arr[n];
    int arr1[n];

    for(int i = 0; i < n; i++){
        cin >> arr[i];
    }
    for(int i = 0; i < n; i++){
        cin >> arr1[i];
    }

    sort(arr, arr+n);
    sort(arr1, arr1+n);

    for(int i = 0; i < n; i++){
        if(arr[i] != arr1[i]){
            cout << "No";
            return 0;
        }
    }
    cout << "Yes";

    return 0;
}